docs/userGuide/syntax/siteNavigationMenus.mbdf

```diff
@@ -3,26 +3,23 @@
 **A _Site Navigation Menu_ (==_siteNav_ for short==) is a fixed menu on the ==left edge== of a page**, that can be used to show a road map of the main pages of your site.
 
 Steps to add a siteNav:
-1. Format your siteNav as an unordered Markdown list and save it inside the `_markbind/navigation` directory.
+1. Format your siteNav as an unordered Markdown list and save it inside the `_markbind/navigation` directory. You may include dividers in the siteNav via <a target="_blank" href="{{ baseUrl }}/userGuide/readerfacingfeatures#horizontal-rules">horizontal rules</a>.
```

Comment:
I feel that we shouldn't open internal links in a new tab. I know that we currently do that in the page and site nav pages, but perhaps we should remove those too.

Reply:
Want me to open in same page or remove link altogether?

Reply:
Let's open it in the same page.